lib/scraper.rb

```diff
@@ -0,0 +1,54 @@
+# frozen_string_literal: true
+
+require 'httparty'
+require 'nokogiri'
+require_relative 'database'
+require_relative 'vacancy_database_handler'
+
+class Scraper
+  BASE_URL = 'https://openai.com'
+  START_URL = 'https://openai.com/careers/search'
+
+  def initialize
+    @database = DataBase.new
```

Comment:
Can you do it outside from this class, in main app class

Reply:
Do you mean to pass a DataBase.new as parameter to initialize method? And make it in a main file? Or maybe i don’t understand you properly

Reply:
def initialize(database, table,  vacancy_handler)
    @database = database
    @table = table
    @vacancy_handler = vacancy_handler
  end

make instances in main file and just pass it as a argument in Scraper.new

maybe something like this ?